qt/scientific_interfaces/ISISReflectometry/test/Reduction/MockGroup.h

```diff
@@ -0,0 +1,52 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "../../../ISISReflectometry/Reduction/Group.h"
+#include <gmock/gmock.h>
```

Comment:
You're missing a few headers in here:
`string`
`vector`
`boost/optional`
And from refl:
`Item.h`

Also `Group.h` should be `IGroup.h`

That way refactoring elsewhere in the codebase won't cause this to randomly break

Reply:
I'll mark this resolved, since it doesn't make sense to hold up the PR for a minor header change